src/Game.qml

```diff
@@ -562,15 +561,24 @@ Item {
         anchors.fill: parent
         visible: false
         opacity: visible ? 0.8 : 0.0
-
-        property int angle: 45
-
-        transform: Rotation {
-            origin.x: width/2
-            origin.y: height/2
-            angle: gameOver.angle
-            Behavior on angle { NumberAnimation { easing.type: Easing.InCubic; duration: 600 } }
-        }
+        radius: DeviceInfo.hasRoundScreen ? Dims.w(50) : 0
+
+        transform: [
+            Rotation {
+                origin.x: width/2
+                origin.y: height/2
+                angle: -45
```

Comment:
Instead of defining a -45 degrees rotation for the gameOver item whose parent is main whose rotation is 45 degrees, I think there should be two items, one for things that are rotated 45 degrees and one for the gameOver screen which would be "naturally" straight, without having to cancel out the rotation of its parent. wdyt?

Reply:
Fixed.